package.json

```diff
@@ -32,6 +32,8 @@
     "bcryptjs": "2.4.3",
     "classnames": "2.3.2",
     "cookies-next": "2.1.1",
+    "crypto-js": "^4.1.1",
+    "daisyui": "^2.51.6",
```

Comment:
What is the need of daisyui here?

Reply:
It is being used in `tailwind.config.js`